config/images.yml

```diff
@@ -117,30 +120,6 @@
             additionalSizes:
                 - {width: 440, height: ~, media: "(max-width: 479px)"}
                 - {width: 730, height: ~, media: "(max-width: 1023px)"}
-        -   name: leftSidebar
-            width: 240
-            height: ~
-            crop: false
-            occurrence: 'Front-end: Advertising in the left panel under the category tree'
-            additionalSizes:
-                - {width: 440, height: ~, media: "(max-width: 479px)"}
-                - {width: 730, height: ~, media: "(max-width: 768px)"}
-        -   name: homepagePrimary
-            width: 628
-            height: 342
-            crop: false
-            occurrence: 'Front-end: Advertising on homepage above popular categories (Primary)'
-        -   name: homepageSecondaryFirst
-            width: 439
-            height: 160
-            crop: false
-            occurrence: 'Front-end: Advertising on homepage above popular categories (Secondary First)'
-        -   name: homepageSecondarySecond
-            width: 439
-            height: 160
-            crop: false
-            occurrence: 'Front-end: Advertising on homepage above popular categories (Secondary Second)'
```

Comment:
And why did you remove this configurations? I don't know if this change will broke some shop functionality?

Reply:
Fixed in 268abd1bae86f02635c43016b0523475fff3663c